Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,30 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+
+//task duration in .... minutes
+const taskDurationMinutes = tasks.map(function(task) {
+  return task.duration;
+})
+
+console.log(taskDurationMinutes); //[ 180, 120, 20, 200, 240, 180, 10, 200, 40 ]
```

Comment:
Excellent work logging as you go.  It is something I still do all the time, and I've been at this for more years than I care to remember. 😉